src/sentry/lang/native/appconnect.py

```diff
@@ -156,10 +156,17 @@ def from_json(cls, data: Dict[str, Any]) -> "AppStoreConnectConfig":
         return cls(**data)
 
     @classmethod
-    def all_for_project(cls, project: Project) -> "List[AppStoreConnectConfig]":
```

Comment:
Looks like this method was left unused after a series of changes addressing feedback. I've replaced this with a similar equivalent because a lot of the checks being done in `inner_refresh_all_builds` resemble the ones being made in other similar conversion class methods.

I needed to break up the original looping logic in `inner_refresh_all_builds` anyways to avoid updating the project setting too often (well, more than once per build refresh).